ipv6_in_real_life/data/de/bank.toml

```diff
@@ -45,5 +50,128 @@ additional_hosts = [
   "www.berliner-sparkasse.de",
   "www.frankfurter-sparkasse.de",
   "www.deka.de",
+  "www.sparkasse-koelnbonn.de",
+  "www.sskduesseldorf.de",
+  "www.f-i.de",
+]
+
+[[bank]]
+main_host = "www.lbb.de"
+additional_hosts = [
+  "stbsk.eu.auth0.com",
+]
+
+[[bank]]
+main_host = "www.lbbw.de"
+additional_hosts = [
+  "www.bw-bank.de",
+]
+
+[[bank]]
+main_host = "ww.vr-bayernmitte.de"
+additional_hosts = [
+  "",
```

Comment:
Where there are no additional hosts, either skip the attribute or leave the list empty (I _think_ the code would error out with the empty string here.)

Reply:
Fixed, thanks.